### PR TITLE
Feat/hedging indicator

### PR DESCRIPTION
**Pull request by clementbalestrat**

### CODE REVIEW

**Comment by clementbalestrat** [comment]
@mi-yu I merged the PR with the lastest changes. Currently working on the design. It should be ready for testing tomorrow :)

**Comment by clementbalestrat** [comment]
@mi-yu alright I've refactored the files a bit. I removed the hedge logic from the AssetsTable. I think the file was a little bit too messy so it's better in its own table component.

It's really looking good now. One last thing though, before we merge it. Would you be able to add the following:

- Add ETH and wETH to the sETH balance.
- Add wBTC, renBTC to the sBTC balance. The idea is to take those into account in case the user sold their synthetic assets to other assets. We still want them to be compared to the debt pool.
- "i" icon with a popup next to sETH and sBTC explaining the above. I'm including a screenshot of the design.

Screen Shot 2021-02-24 at 4 09 49 pm

**Comment by mi-yu** [comment]
Yep I can work on this!

File: pages/debt.tsx [hunk 1]
```diff
@@ -1,22 +1,20 @@
 import React from 'react';
 import Head from 'next/head';
 import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { last } from 'lodash';
```
Comment (evgenyboxer) [inline pages/debt.tsx:5]:
```suggestion
import last from 'lodash/last';
```

File: queries/synths/useSynthsTotalSupplyQuery.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+			const supplyData: SynthTotalSupply[] = [];
+			for (let i = 0; i < synthTotalSupplies[0].length; i++) {
+				const value = toBigNumber(utils.formatEther(synthTotalSupplies[2][i]));
+				const name = utils.parseBytes32String(synthTotalSupplies[0][i]);
+				const totalSupply = toBigNumber(utils.formatEther(synthTotalSupplies[1][i]));
+
+				let combinedWithShortsValue = value;
+				if (name === 'iETH') {
+					combinedWithShortsValue = combinedWithShortsValue.plus(ethShorts.times(ethPrice));
+				} else if (name === 'iBTC') {
```
Comment (evgenyboxer) [inline queries/synths/useSynthsTotalSupplyQuery.ts:71]: ^ minor - please use `Synths.iBTC` etc

File: queries/walletBalances/useWETHBalanceQuery.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+const useWETHBalanceQuery = (options?: QueryConfig<BigNumber>) => {
+	const { provider } = Connector.useContainer();
+	const isWalletConnected = useRecoilValue(isWalletConnectedState);
+	const walletAddress = useRecoilValue(walletAddressState);
+	const network = useRecoilValue(networkState);
+
+	return useQuery<BigNumber>(
+		QUERY_KEYS.WalletBalances.WETH(walletAddress ?? '', network?.id!),
+		async () => {
+			const contract = new ethers.Contract(
```
Comment (evgenyboxer) [inline queries/walletBalances/useWETHBalanceQuery.ts:23]: same comment as above, this needs to be outside the function, possibility - we just put it on `lib/synthetix.ts`?

File: sections/debt/components/DebtPieChart/DebtPieChart.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+		const remainingSupply = { ...remaining[0] };
+		remainingSupply.name = 'Other';
+		for (const data of remaining.slice(1)) {
+			remainingSupply.value = remainingSupply.value.plus(data.value);
+		}
+		remainingSupply.poolProportion = remainingSupply.value.div(totalSupply?.totalValue ?? 0);
+		topNSynths.push(remainingSupply);
+	}
+
+	pieData = topNSynths
```
Comment (evgenyboxer) [inline sections/debt/components/DebtPieChart/DebtPieChart.tsx:72]: this could use `useMemo`

**Review by evgenyboxer — APPROVED** [review]

**Pull request merged**